initramfs/init.sh

```diff
@@ -438,6 +432,7 @@ toolcheck
 kernel_mount
 parse_cmdline
 enable_usb_storage_device
+# TODO re-enable when screen is attached
```

Comment:
Yes, we should somehow make sure that we re-enable this once we all have displays attached to our ultimainboards. I'll ping Miguel about it

Reply:
Yes or I can test it on the target, but I think it would be good to separate the user feedback and splash screens to a different ticket.